.github/workflows/pr-tests.yml

```diff
@@ -15,8 +15,7 @@ concurrency:
 
 env:
   CARGO_TERM_COLOR: always
-  POWDR_GENERATE_PROOFS: "true"
-  MAX_DEGREE_LOG: "20"
+  POWDR_OPENVM_SEGMENT_DELTA: 10000
```

Comment:
What about this one?

Reply:
we need this env var for a hacky solution we have when an apc runs too close to the segment boundary in openvm. We have this same env var in the nightly tests. The bug doesn't happen in pr-tests right now but good to have the env var set already if we ever add larger tests to pr-tests